cmd/minikube/cmd/config/configure.go

```diff
@@ -234,6 +234,24 @@ var addonsConfigureCmd = &cobra.Command{
 			if err := config.SaveProfile(profile, cfg); err != nil {
 				out.ErrT(style.Fatal, "Failed to save config {{.profile}}", out.V{"profile": profile})
 			}
+		case "registry-aliases":
+			profile := ClusterFlagValue()
+			_, cfg := mustload.Partial(profile)
+			validator := func(s string) bool {
+				format := regexp.MustCompile(`^([a-zA-Z0-9-_]+\.[a-zA-Z0-9-_]+)+(\ [a-zA-Z0-9-_]+\.[a-zA-Z0-9-_]+)*$`)
+				return format.MatchString(s)
+			}
+			registryAliases := AskForStaticValidatedValue("-- Enter registry aliases separated by space: ", validator)
+			cfg.KubernetesConfig.RegistryAliases = registryAliases
+
+			if err := config.SaveProfile(profile, cfg); err != nil {
+				out.ErrT(style.Fatal, "Failed to save config {{.profile}}", out.V{"profile": profile})
+			}
+
+			// Re-enable registry-aliases addon in order to generate template manifest files with custom hosts
```

Comment:
I would only enable the addon if it were already enabled. Otherwise, let the user do it themselves.

Reply:
Makes sense.